Client/core/CCore.cpp

```diff
@@ -124,12 +124,19 @@ CCore::CCore()
 
     // Create tray icon
     m_pTrayIcon = new CTrayIcon();
+
+    // Initialize discord manager
+    m_pDiscordManager = new CDiscordManager();
```

Comment:
Please use a `std::unique_ptr` for new things.

Reply:
In this case `CDiscordManager` should just be a member of `CCore`. There is no need to use a pointer here at all.

Reply:
Alright.

Reply:
I think a pointer is better here because it avoids dependancy on CDiscordManager.h

Reply:
Would that still be the case with the fwd decl mentioned [here](https://github.com/multitheftauto/mtasa-blue/pull/1031#discussion_r307152804)?

Reply:
Yes. The fwd decl mentioned [there](https://github.com/multitheftauto/mtasa-blue/pull/1031#discussion_r307152804) requires a pointer

Reply:
So class CDiscordManager; and no more including the header in CCore.h

But I still need to go with smart pointers right?